wiki/Ranking_Criteria/osu!mania/es.md

```diff
@@ -0,0 +1,205 @@
+- **Trucos que involucren la velocidad del slider deben ser reaccionables entre un 1/2 de una pulsación o más**
+
+Pauta adiciónales para dificultades *Hard de 4 teclas*:
+
+- **Evite chords con más de 2 notas.** Esto incluye patrones de notas largas que involucren más de tres columnas.
+- **Evite los anchors de cinco o más notas.** A la vez que son fáciles de leer para los jugadores, anchors largos son agotadores.
+
+Pautas adiciónales para dificultades *Hard de 7 teclas*:
+
+- **Evite los anchors de cuatro o más notas.** Esto incluye patrones de notas largas que involucren más de cuatro columnas.
+- **Evite los anchors de cinco o más notas.**  A la vez que son fáciles de leer para los jugadores, anchors largos son agotadores.
+- **Mantenga la distancia de los chords en un chordstream en 1/1 o intervalo más grande.** Algo más denso que esto puede afectar la lectura del beatmap.
+- **Hands o triples ocasionales pueden ser usados en streams, siempre cuando estén separados por al menos un compás (4 pulsaciones).**
+
+### ![](/wiki/shared/diff/insane-m.png?20211215) Insane
+
+#### Pautas
+
+- **Evite usar picos de dificultad no justificados.** La dificultad debería ser representativa sobre la intensidad de la canción.
+- **Evite usar split rolls o cualquier tipo de patrón complicado de streams 1/8 que dure más de 4 pulsaciones o un compás.** Streams de este tipo de snap son difíciles de ejecutar y solo deberían ser utilizados si la canción lo amerita.
+- **Evite usar streams de notas largas de 1/6 o más** Estos son complicados de jugar debido a su corta duración. Use notas normales en los streams en vez de notas largas.
+- **Cambios largos de la velocidad del slider deben ser entre 0.7x y 1.10x.**
+- **Trucos que involucren la velocidad del slider deben ser reaccionables entre un 1/4 de una pulsación o más cuando no están normalizados y 1/2 cuando están normalizados.**
+
+Pautas adiciónales para dificultades *Insane de 4 teclas*:
+
+- **Evite usar chords en el medio de un stream 1/8.** La densidad de estos patrones es usualmente muy difícil para jugadores de este nivel, especialmente cuando se es usado en dificultades bajas de un beatmap. Adicionalmente, la colocación no efectiva de estos chords puede resultar en el uso de minijacks de 1/4. Usar un chord al principio o final de un stream está bien.
+- **Usar split jumptrills por más de 2 pulsaciones que terminen en un hand es desalentado.** A no ser que el beatmap use jacks u otro patrón frecuentemente, este patrón es difícil de jugar. Si un hand es necesario, borre la nota antes la cual crea el mini jack.
+- **Evite usar más de 9 notas en un split jumptrill.** Estos son más físicamente extenuantes que otros patrones con densidad similar.
+
+Pautas adiciónales para dificultades *Insane de 7 teclas*:
+
+- **Evite usar jacks de 1/4 con más de 3 notas en el medio de un chordstream.** Esto requiere una independencia de los dedos que estos jugadores de esta dificultad no pueden manejar.

```

Comment:
```suggestion
- **Evite el uso de jacks de 1/4 con 3+ notas en medio del chordstream.** Estos requieren una independencia de los dedos que es demasiado difícil para los jugadores de este nivel.
```

Reply:
Added "el uso de" kept everything else do to the coherence thing I mentioned above with the same guideline